packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -184,6 +184,11 @@ const cli = {
         describe: 'Whether or not to automatically update all reference screenshots with the latest screenshots.',
         default: false,
       },
+      useRemoteScreenshots: {
```

Comment:
I'm open for other name suggestions.

Reply:
Do you think we'll need to add flags to view remote diffs, or remote latest? If so - I suggest naming it `useRemoteRefScreenshots` so we have room to add those flags later. If not, or if we don't know, this name looks fine to me.

Reply:
I don't see us needing additional options for diff/latest screenshots since we are storing only reference screenshots. But I like your suggestion of `useRemoteRefScreenshots` to be more explicit. Do you prefer `useRemoteRefScreenshots` or `useRemoteReferenceScreenshots`?

Reply:
I prefer `useRemoteReferenceScreenshots` with a shorthand flag like `-R` if that's possible

Reply:
I've updated to name to `useRemoteReferenceScreenshots1`: https://github.com/cerner/terra-toolkit/pull/732/commits/aa66ee6480d4eadfde686aea0efbb0c0723c3d50

We previously only created the `u` alias for `updateScreenshots`. The reason was to mimic jest in that you can run  `npm run jest -- -u` given the occasional need to _update_ the jet snapshots on the fly. So there is value in creating the `u` alias so users don't need to type `updateScreenshots` every time in the terminal. 

However, the use case for `useRemoteReferenceScreenshots` is different in that once consumers opt in for this feature by setting it in their wdio test scripts in package.json they should not be flipping this option on and off, hence an alias provides little value. It is expected that when set and opted in they should be discouraged to not be going back to the old way of storing screenshots locally.